apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.html.ts

```diff
@@ -71,66 +75,63 @@ export function InstructorAssessmentStatistics({
 
           <div class="card mb-4">
             <div class="card-header bg-primary text-white">
-              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Score statistics
+              ${resLocals.assessment_set.name} ${assessment.number}: Score statistics
             </div>
-            ${resLocals.assessment.score_stat_number > 0
+            ${assessment.score_stat_number > 0
               ? html`
                   <div class="card-body">
-                    <div id="scoreHist" class="histogram"></div>
-                    <script>
-                      $(function () {
-                        var data = [${escapeHtml(resLocals.assessment.score_stat_hist)}];
-                        var xgrid = _.range(0, 110, 10);
-                        var options = {
-                          ymin: 0,
-                          xlabel: 'score / %',
-                          ylabel: 'number of students',
-                        };
-                        histogram('#scoreHist', data, xgrid, options);
-                      });
-                    </script>
+                    <div
+                      id="scoreHist"
```

Comment:
Is this ID still needed?

Reply:
No, removed this and other unused IDs in [af7f4c3](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/af7f4c3e9647b371816baf4e203c7f77672e059e).